capstone-frontend/src/styles/ListPage.css

```diff
@@ -1,3 +1,12 @@
+#modal-buttons {
+  display: flex;
+  float: right
+}
+
+.margin-auto {
+  margin: auto;
+}
+
```

Comment:
Instead of using auto, try using `mx-auto` in your element classNames.

https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
Oh, I didn't know that bootstrap provided an auto margin. I think for this scenario, I also need the y margin set to auto.

Reply:
Then could you use mx-auto and my-auto together?